src/style.css

```diff
@@ -184,3 +186,123 @@ header {
     margin-bottom: 5px;
   }
 }
+
+.breed-entry {
+  margin: 50px;
+  height: 400px;
+  width: 350px;
+  padding: 10px;
+  box-sizing: border-box;
+  border: 1px lightgray solid;
+  border-radius: 5px;
+}
+
+.breed-picture {
+  width: 330px;
+  height: auto;
+}
+
+/* Popup styling */
+article,
+div {
+  box-sizing: border-box;
+}
+
+p {
+  margin: 0;
+}
+
+#popup-article {
+  background-color: rgba(0, 0, 0, 0.7);
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+#popup-container {
+  background-color: white;
+  height: 90vh;
+  width: 70vw;
+  max-width: 650px;
+  margin: 5vh auto;
+  border: 1px lightgray solid;
+  border-radius: 15px;
+  overflow: hidden;
+}
+
+.popup-scroll {
+  width: 100%;
+  height: 100%;
+  padding: 20px;
+  overflow: hidden;
+  overflow-y: scroll;
+}
+
+#close-popup {
+  float: right;
+  padding: 0;
+  border: none;
+  background: none;
+}
+
+.popup-image {
+  height: 300px;
+  max-width: 610px;
+  display: block;
+  margin: auto;
+}
+
+h2 {
+  margin: 10px;
+  font-size: 30px;
+  font-weight: bold;
+  text-align: center;
+}
+
+h3 {
+  margin: 10px;
+  font-size: 20px;
+  font-weight: bold;
+  text-align: center;
+}
+
+.info-row {
+  padding: 8px 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.info-entry-label {
+  font-weight: bold;
+}
+
+#comments-list {
+  list-style: none;
+}
+
+.comment span {
+  display: inline-block;
+  padding-right: 5px;
+}
+
+#new-comment {
+  width: 70%;
+  margin: auto;
+}
+
+#new-comment-name {
+  width: 100%;
+}
+
+#new-comment-content {
+  width: 100%;
+  height: 70px;
+}
+
+#new-comment-submit {
+  display: block;
+  margin: auto;
+  padding: 5px 10px;
+}
```

Comment:
Great work @notlfish everything is clean and coherent, the only question I have is, this code is responsive?, I didn't see it on mobile, but I think it's not a requirement, we can go through this later. :+1:

Reply:
It certainly isn't. I don't see how the code in that snippet, in particular, could be a problem, but the whole popup is not responsive.
Since it's not in the project requirements, I won't add responsive styling until I finish with everything else. I can open an issue to do it after everything else is in place, though.